InnaFeature/Steps/InteractionsSteps.cs

```diff
@@ -0,0 +1,75 @@
+﻿using InnaFeature.Helpers.Browser;
+using InnaFeature.Pages;
+using NUnit.Framework;
+
+namespace InnaFeature.Steps
+{
+    [Binding]
+    internal class InteractionsSteps
+    {
+        private readonly BasePage basePage;
+        private readonly Interactions interactions;
+        private readonly IBrowserHelper browserHelper;
+
+        public InteractionsSteps(IBrowserHelper browserHelper)
+        {
+            this.browserHelper = browserHelper;
+            basePage = new BasePage(browserHelper);
+            interactions = new Interactions(browserHelper);
+        }
+
+        [Given(@"User is on the ""([^""]*)"" homepage")]
+        public void GivenUserIsOnTheDemoqa_ComHomepage(string applicationUnderTestUrl)
+        {
+            browserHelper.WebDriver.Navigate().GoToUrl(applicationUnderTestUrl);
+        }
+
+        [Given(@"User navigates to the category ""([^""]*)""")]
+        public void NavigateToTheInteractionsCategory(string category)
+        {
+            basePage.NavigateToTheCategory(category);
+        }
+
+        [When(@"User navigates to section ""([^""]*)""")]
+        public void NavigateToSelectableSection(string section)
+        {
+            basePage.NavigateToTheSection(section);
+        }
+
+        [When(@"User clicks on the ""([^""]*)"" tab")]
+        public void NavigateToGridTab()
+        {
+            interactions.GridTab.Click();
+        }
+
+        [When(@"User selects squares (.*), (.*), (.*), (.*), and (.*)")]
+        public void SelectSquares()
```

Comment:
why do you have parameters in the step definition, but your method assigned to that step does not have any acceptable parameters?